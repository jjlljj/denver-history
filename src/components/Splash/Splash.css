
#splash-page {
  background-image: url('./background_min.jpg');
  text-align: center; 
  height: 100vh;
  position: relative;
  width: 100vw;
  background-repeat: no-repeat;
  background-size: cover;
  margin-top: 0px;
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
}

#layer {
  background: linear-gradient(to right, rgba(63, 81, 181, .67), rgba(86, 179, 224, .9)); 
  position: absolute;
  top: 0;
  left: 0;
  width: 100vw;
  height: 100vh;
}

.logo {
  color: #f6f6f4;
  font-size: 64px;
  font-family: 'Milonga', cursive;
  font-weight: 700;
  margin-top: 30vh;
  margin-bottom: 10vh;
  text-shadow: 2px 2px #291845;
}

#tour-button {
  background-color: #281845;
  box-shadow: 5 5 5 5 #ccc;
  color: #f6f6f4;
  display: inline-block;
  font-weight: 700;
  letter-spacing: 1px;
  margin: auto;
  padding: 15px 25px;
}

#tour-button:hover {
  background-color: #4c3672;
}

#tour-button:active {
  background-color: #bda8ad;
}
