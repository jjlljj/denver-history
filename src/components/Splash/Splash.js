import React, { Component } from 'react';
import PropTypes from 'prop-types';
// import './Splash.css';

export default class Splash extends Component {
  render() {
    return (
      <div>
        <h1>Historic Denver SPLASH</h1>
      </div>
    )
  }
}