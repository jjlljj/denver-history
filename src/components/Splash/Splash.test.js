/* eslint-disable */
import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import Splash from './Splash';

describe('Splash', () => {

  it('matches snapshot', () => {
    const renderedComponent = shallow(<Splash />)
    expect(renderedComponent).toMatchSnapshot()
  })
})
