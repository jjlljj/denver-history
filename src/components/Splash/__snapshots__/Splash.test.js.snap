// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Splash matches snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Splash />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        id="layer"
>
        <h1
                class="logo"
        >
                Historic Denver
        </h1>
        <Link
                replace={false}
                to="/tour"
        >
                <div
                        id="tour-button"
                >
                        Enter Tour
                </div>
        </Link>
</div>,
      "id": "splash-page",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1
            class="logo"
>
            Historic Denver
</h1>,
          <Link
            replace={false}
            to="/tour"
>
            <div
                        id="tour-button"
            >
                        Enter Tour
            </div>
</Link>,
        ],
        "id": "layer",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Historic Denver",
            "class": "logo",
          },
          "ref": null,
          "rendered": "Historic Denver",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <div
              id="tour-button"
>
              Enter Tour
</div>,
            "replace": false,
            "to": "/tour",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Enter Tour",
              "id": "tour-button",
            },
            "ref": null,
            "rendered": "Enter Tour",
            "type": "div",
          },
          "type": [Function],
        },
      ],
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          id="layer"
>
          <h1
                    class="logo"
          >
                    Historic Denver
          </h1>
          <Link
                    replace={false}
                    to="/tour"
          >
                    <div
                              id="tour-button"
                    >
                              Enter Tour
                    </div>
          </Link>
</div>,
        "id": "splash-page",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h1
              class="logo"
>
              Historic Denver
</h1>,
            <Link
              replace={false}
              to="/tour"
>
              <div
                            id="tour-button"
              >
                            Enter Tour
              </div>
</Link>,
          ],
          "id": "layer",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Historic Denver",
              "class": "logo",
            },
            "ref": null,
            "rendered": "Historic Denver",
            "type": "h1",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": <div
                id="tour-button"
>
                Enter Tour
</div>,
              "replace": false,
              "to": "/tour",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Enter Tour",
                "id": "tour-button",
              },
              "ref": null,
              "rendered": "Enter Tour",
              "type": "div",
            },
            "type": [Function],
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
